src/platform/forms/save-in-progress/RoutedSavableApp.jsx

```diff
@@ -102,9 +105,7 @@ class RoutedSavableApp extends React.Component {
       newProps.prefillStatus !== this.props.prefillStatus &&
       newProps.prefillStatus === PREFILL_STATUSES.unfilled
     ) {
-      newProps.router.push(
-        newProps.routes[this.props.routes.length - 1].pageList[1].path,
```

Comment:
This was somewhat annoying to debug... Before this last set of changes, "Start Application" as a logged in user kept taking me to `/id-form` because it was still the second page in the list (the overall list, not the list of "active" or "eligible" pages). And then there's handling in `IDPage` to redirect back to the intro, so attempting to start the application would just bring me back to the intro, despite how the `depends` should have skipped the ID form.